[BUG]测试示例 - HDFS Client文件访问加速 无法通过回归测试

**What is your environment(Kubernetes version, Fluid version, etc.)**

kubernetes version：
Client Version: version.Info{Major:"1", Minor:"16+", GitVersion:"v1.16.9-aliyun.1", GitCommit:"4f7ea78", GitTreeState:"", BuildDate:"2020-05-08T07:35:38Z", GoVersion:"go1.13.9", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"16+", GitVersion:"v1.16.9-aliyun.1", GitCommit:"4f7ea78", GitTreeState:"", BuildDate:"2020-05-08T07:29:59Z", GoVersion:"go1.13.9", Compiler:"gc", Platform:"linux/amd64"}

fluid version：
BuildDate: 2020-11-01_15:00:37
GitCommit: 1c40aee

**Describe the bug**
测试示例 - HDFS Client文件访问加速 无法通过回归测试
fluid-hdfs-demo报错日志为：
2020-11-03 15:39:34 WARN  NativeCodeLoader:60 - Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
2020-11-03 15:39:34 INFO  AbstractFileSystem:481 - Creating Alluxio configuration from Hadoop configuration {}, uri configuration {alluxio.zookeeper.address=null, alluxio.zookeeper.enabled=false, alluxio.master.hostname=hadoop-master-0.default.svc.cluster.local, alluxio.master.rpc.addresses=null, alluxio.master.embedded.journal.addresses=null, alluxio.master.rpc.port=19998}
2020-11-03 15:39:34 INFO  AbstractFileSystem:501 - Initializing filesystem with connect details hadoop-master-0.default.svc.cluster.local:19998
2020-11-03 15:39:34 INFO  TieredIdentityFactory:63 - Initialized tiered identity TieredIdentity(node=fluid-hdfs-demo-6dp6d, rack=null)
2020-11-03 15:39:34 INFO  NettyUtils:147 - EPOLL_MODE is available
Exception in thread "main" alluxio.exception.status.UnavailableException: Failed to connect to FileSystemMasterClient @ hadoop-master-0.default.svc.cluster.local:19998 after 44 attempts
	at alluxio.AbstractClient.connect(AbstractClient.java:278)
	at alluxio.client.file.BaseFileSystem.rpc(BaseFileSystem.java:518)
	at alluxio.client.file.BaseFileSystem.getStatus(BaseFileSystem.java:258)
	at alluxio.client.file.BaseFileSystem.openFile(BaseFileSystem.java:334)
	at alluxio.client.file.FileSystem.openFile(FileSystem.java:422)
	at alluxio.hadoop.HdfsFileInputStream.<init>(HdfsFileInputStream.java:60)
	at alluxio.hadoop.AbstractFileSystem.open(AbstractFileSystem.java:627)
	at org.apache.hadoop.fs.FileSystem.open(FileSystem.java:945)
	at io.fluid.demo.HDFSClient.readFile(HDFSClient.java:32)
	at io.fluid.demo.HDFSClient.main(HDFSClient.java:21)
Caused by: alluxio.exception.status.UnavailableException: Failed to handshake with master hadoop-master-0.default.svc.cluster.local:19998 to load cluster default configuration values: UNAVAILABLE: io exception
	at alluxio.util.ConfigurationUtils.loadConfiguration(ConfigurationUtils.java:504)
	at alluxio.ClientContext.loadConf(ClientContext.java:134)
	at alluxio.ClientContext.loadConfIfNotLoaded(ClientContext.java:155)
	at alluxio.AbstractClient.beforeConnect(AbstractClient.java:175)
	at alluxio.AbstractClient.connect(AbstractClient.java:224)
	... 9 more
Caused by: io.grpc.StatusRuntimeException: UNAVAILABLE: io exception
	at io.grpc.stub.ClientCalls.toStatusRuntimeException(ClientCalls.java:240)
	at io.grpc.stub.ClientCalls.getUnchecked(ClientCalls.java:221)
	at io.grpc.stub.ClientCalls.blockingUnaryCall(ClientCalls.java:140)
	at alluxio.grpc.MetaMasterConfigurationServiceGrpc$MetaMasterConfigurationServiceBlockingStub.getConfiguration(MetaMasterConfigurationServiceGrpc.java:384)
	at alluxio.util.ConfigurationUtils.loadConfiguration(ConfigurationUtils.java:498)
	... 13 more
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: finishConnect(..) failed: Connection refused: hadoop-master-0.default.svc.cluster.local/192.168.1.147:19998
Caused by: java.net.ConnectException: finishConnect(..) failed: Connection refused
	at io.netty.channel.unix.Errors.throwConnectException(Errors.java:124)
	at io.netty.channel.unix.Socket.finishConnect(Socket.java:251)
	at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.doFinishConnect(AbstractEpollChannel.java:672)
	at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.finishConnect(AbstractEpollChannel.java:649)
	at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.epollOutReady(AbstractEpollChannel.java:529)
	at io.netty.channel.epoll.EpollEventLoop.processReady(EpollEventLoop.java:465)
	at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:378)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:989)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at java.base/java.lang.Thread.run(Unknown Source)

**What you expect to happen:**

**How to reproduce it**

**Additional Information**

--- a/docs/en/samples/accelerate_data_accessing_by_hdfs.md
+++ b/docs/en/samples/accelerate_data_accessing_by_hdfs.md
@@ -170,7 +170,7 @@ At the same time, add the configuration of alluxio in core-site.xml. For details
 When accessing files through the HDFS client, you need to specify the HDFS server address
 
 ```java
-final String HDFS_URL = "alluxio://hadoop-master-0.default:19998/hadoop"
+final String HDFS_URL = "alluxio://hadoop-master-0.default.svc.cluster.local:"+ System.getenv("HADOOP_PORT") + "/hadoop";
 Configuration conf = new Configuration();
 FileSystem fs = FileSystem.get(URI.create(HDFS_URL), conf);
 ```
@@ -189,7 +189,7 @@ NAME    UFS TOTAL SIZE   CACHED   CACHE CAPACITY   CACHED PERCENTAGE   PHASE   H
 hbase   443.49MiB        0.00B    4.00GiB          0.0%                Bound   alluxio://hbase-master-0.default:19998   97s
 ```
 
-For the complete test code, please refer to [samples/hdfs](../../../samples/hdfs). We made the test code into a mirror to facilitate the next test. The mirror address is registry.cn-beijing.aliyuncs.com/yukong/fluid-hdfs-demo:1.0.0.
+For the complete test code, please refer to [samples/hdfs](../../../samples/hdfs). We made the test code into a mirror to facilitate the next test. The mirror address is registry.cn-hangzhou.aliyuncs.com/qiulingwei/fluid-hdfs-demo:1.2.0
 
 **查看待创建的测试作业**
 
@@ -205,10 +205,14 @@ spec:
       restartPolicy: OnFailure
       containers:
         - name: fluid-hdfs-demo
-          image: registry.cn-beijing.aliyuncs.com/yukong/fluid-hdfs-demo:1.0.0 #可以替换为自己构建的镜像
+          image: registry.cn-hangzhou.aliyuncs.com/qiulingwei/fluid-hdfs-demo:1.2.0
           imagePullPolicy: Always
+          env:
+          - name: HADOOP_PORT
+            value: "19998"
 EOF
 ```
+Here, you need to replace 19998 in the environment variable with the actual port in the HCFS (Hadoop Compatible FileSystem) URL just queried
 
 **启动测试作业**
 

--- a/docs/zh/samples/accelerate_data_accessing_by_hdfs.md
+++ b/docs/zh/samples/accelerate_data_accessing_by_hdfs.md
@@ -170,7 +170,7 @@ Dataset资源对象准备完成后（即与Alluxio实例绑定后），与该资
 在通过HDFS Client访问文件时，需要指定HDFS Server地址
 
 ```java
-final String HDFS_URL = "alluxio://hadoop-master-0.default:19998/hadoop"
+final String HDFS_URL = "alluxio://hadoop-master-0.default.svc.cluster.local:"+ System.getenv("HADOOP_PORT") + "/hadoop";
 Configuration conf = new Configuration();
 FileSystem fs = FileSystem.get(URI.create(HDFS_URL), conf);
 ```
@@ -189,7 +189,7 @@ NAME    UFS TOTAL SIZE   CACHED   CACHE CAPACITY   CACHED PERCENTAGE   PHASE   H
 hbase   443.49MiB        0.00B    4.00GiB          0.0%                Bound   alluxio://hbase-master-0.default:19998   97s
 ```
 
-完整的测试代码可参考[samples/hdfs](../../../samples/hdfs)。我们把测试代码制作成镜像，方便接下来的测试，镜像地址为 registry.cn-beijing.aliyuncs.com/yukong/fluid-hdfs-demo:1.0.0。
+完整的测试代码可参考[samples/hdfs](../../../samples/hdfs)。我们把测试代码制作成镜像，方便接下来的测试，镜像地址为registry.cn-hangzhou.aliyuncs.com/qiulingwei/fluid-hdfs-demo:1.2.0
 
 **查看待创建的测试作业**
 
@@ -205,10 +205,14 @@ spec:
       restartPolicy: OnFailure
       containers:
         - name: fluid-hdfs-demo
-          image: registry.cn-beijing.aliyuncs.com/yukong/fluid-hdfs-demo:1.0.0 #可以替换为自己构建的镜像
+          image: registry.cn-hangzhou.aliyuncs.com/qiulingwei/fluid-hdfs-demo:1.2.0
           imagePullPolicy: Always
+          env:
+          - name: HADOOP_PORT
+            value: "19998"
 EOF
 ```
+此处，需要将环境变量中的19998替换为刚刚查询得到的HCFS(Hadoop Compatible FileSystem) URL中实际的端口
 
 **启动测试作业**
 

--- a/samples/hdfs/fluid/app.yaml
+++ b/samples/hdfs/fluid/app.yaml
@@ -8,5 +8,8 @@ spec:
       restartPolicy: OnFailure
       containers:
         - name: fluid-hdfs-demo
-          image: registry.cn-beijing.aliyuncs.com/yukong/fluid-hdfs-demo:1.0.0
+          image: registry.cn-hangzhou.aliyuncs.com/qiulingwei/fluid-hdfs-demo:1.2.0
           imagePullPolicy: Always
+          env:
+          - name: HADOOP_PORT
+            value: "20000"

--- a/samples/hdfs/src/main/java/io/fluid/demo/HDFSClient.java
+++ b/samples/hdfs/src/main/java/io/fluid/demo/HDFSClient.java
@@ -10,7 +10,7 @@ import java.net.URI;
 
 public class HDFSClient {
 
-    private static final String HDFS_URL = "alluxio://hadoop-master-0.default.svc.cluster.local:19998/hadoop";
+    private static final String HDFS_URL = "alluxio://hadoop-master-0.default.svc.cluster.local:"+ System.getenv("HADOOP_PORT") + "/hadoop";
     private static final String FILE_PATH = "/hadoop/RELEASENOTES.md";
     private static final String BASE_PATH = "/hadoop/";
 

